apm-dist/release-docs/NOTICE

```diff
@@ -892,3 +892,12 @@ the 'license' directory of the distribution file, for the license terms of the
 components that this product depends on.
 
 ------
+
+========================================================================
+Freemarker NOTICE
+========================================================================
+Apache FreeMarker
+Copyright 2015-2018 The Apache Software Foundation
+
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
```

Comment:
Apache notice does not need to add. We are in the same foundation, read the first lines of this file.